src/foodbox.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+function Foodbox(props) {
+  const [quantity, setQuantity] = useState(props.food.quantity);
+
+  const handleQuantity = (event) => {
+    event.preventDefault();
```

Comment:
Again, you don't need `event.preventDefault()` here since you're not working with a form element.